spec/mailers/admin_mailer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
```

Comment:
Hmm, I think this spec is for the feature we removed in #2941. I'm a little confused why it's passing.